FreeCite.js

```diff
@@ -0,0 +1,76 @@
+{
+	"translatorID": "7bbef39f-8bb9-44d7-826f-47ce75eb15ae",
+	"label": "FreeCite",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://freecite\\.library\\.brown\\.edu/citations/create",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-03-15 22:54:52"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2015 Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (doc.getElementsByTagName("code").length) {
+		return "multiple";
+	}
+}
+
+function doWeb(doc, url) {
+	var texts = [], text = "";
+	var codes = doc.getElementsByTagName("code");
```

Comment:
FYI, the page seems to always contain a reference `<code>` block at the bottom that just has `ContextObject` in it. You want to avoid picking that up.